deployment/docker-compose.yml

```diff
@@ -102,6 +102,8 @@ services:
     ports:
       - "8080:8080"
     volumes:
+      - geoserver/config.yaml:/usr/local/tomcat/webapps/geoserver/data/printing/config.yaml
+      - geoserver/WAPP_logo.png:/usr/local/tomcat/webapps/geoserver/data/printing/WAPP_logo.png
```

Comment:
I think a safe method would be to add these files inside the image and modify entrypoint https://github.com/kartoza/Woolpert/blob/main/src/entrypoint.sh#L100 to copy the files on runtime to the geoserver_data directory

something like 

```
#!/usr/bin/env bash
cp -r /data/WAPP_logo.png ${GEOSERVER_DATA_DIR}/printing
cp -r /data/config.yaml ${GEOSERVER_DATA_DIR}/printing

```

then run /bin/bash foo.sh which will copy the files because on Kubernetes we do not mount volumes with data